test_tools/util.py

```diff
@@ -5,6 +5,7 @@
 from requests.exceptions import ConnectionError
```

Comment:
Try to find some downloadable big file, e.g. in NaturalEarth, and use it instead of africa_surface_lithology.zip